content/blog/2023-09-07-making-connecting-to-localstack-easier/index.md

```diff
@@ -0,0 +1,184 @@
+---
+title: How we are making connecting to LocalStack easier
+date: 2023-09-07
+lastmod: 2023-09-26
+tags:
+- news
+contributors:
+- Simon Walker
+- Daniel Fangl
+---
+
+Connecting your application code to LocalStack is not always easy.
```

Comment:
Can you maybe add a note why is that so (just a line or two)? This would give first-time readers some context about the problem that is being solved :)